lib/install/management/store_files.go

```diff
@@ -139,6 +139,12 @@ func (d *Dispatcher) deleteDatastoreFiles(ds *object.Datastore, path string, for
 	return true, nil
 }
 
+func (d *Dispatcher) geVC65() bool {
+	vcVersion := d.session.Client.ServiceContent.About.Version
+	// vsphere only has versions 5.5.x, 6.0.x, 6.5.x, not likely to have 6.10.x, so compare string directly here
+	return strings.Compare(vcVersion, "6.5.0") >= 0
```

Comment:
The only reason for this method is to select a difference in a specific behaviour. If this is being broken out into a separate method then it should be something like `deleteRequiresEmptyDirectory()` that actually describes the use. Otherwise we'll end up with `geVC70` as well once 6.5 is released and we start testing against vmkernel main.